service/matching/pri_task_writer.go

```diff
@@ -185,6 +187,9 @@ func (w *priTaskWriter) taskWriterLoop() {
 
 	var reqs []*writeTaskRequest
 	for {
+		atomic.StoreInt64(&w.lastTaskIDBlock.start, w.taskIDBlock.start)
```

Comment:
we could use an atomic.Pointer to a taskIDBlock to eliminate the possibility of skew on read, but it's already better than before, and this is just informative, so it's fine with me.

if you think about it, the `end` is already redundant info, it can always be computed from the start. so we could refactor everything to use one value. but we don't need to bother with that now.

Reply:
Oh, right; using `RangeSize`? I see it's a static value (the only one in the config, it seems). That would be simper (assuming we don't make that dynamically configurable in the future).